docs/docs/Get-Started/get-started-installation.md

```diff
@@ -6,166 +6,228 @@ slug: /get-started-installation
+3. To change to the selected version, click **Confirm**.
+The application restarts with the new version installed.
 
-</TabItem>
-</Tabs>
+## Install and run Langflow with Docker {#install-and-run-langflow-docker}
 
-3. To confirm that a local Langflow instance starts, go to the default Langflow URL at `http://127.0.0.1:7860`.
+[Docker](https://docs.docker.com/) provides the simplest way to run Langflow in an isolated environment to ensure consistent behavior across different systems and eliminate dependency conflicts.
+The Langflow Docker image is available on [Docker Hub](https://hub.docker.com/r/langflowai/langflow).
 
-After confirming that Langflow is running, create your first flow with the [Quickstart](/get-started-quickstart).
+1. With Docker installed and running on your system, run this command to pull the `latest` Langflow image and start it:
 
-### Manage Langflow OSS versions
+    ```bash
+    docker run -p 7860:7860 langflowai/langflow:latest
+    ```
 
-To upgrade Langflow to the latest version, use one of the following commands:
+2. To start using Langflow, visit `http://localhost:7860/`.
 
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+For more information, see [Deploy Langflow on Docker](/deployment-docker).
 
-```bash
-uv pip install langflow -U
-```
+## Install and run Langflow OSS
 
-</TabItem>
-<TabItem value="pip" label="pip">
+Before you install and run Langflow OSS, be sure you have the following items.
 
-```bash
-pip install langflow -U
-```
+- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/)
+- [uv](https://docs.astral.sh/uv/getting-started/installation/)
+- A dual-core CPU and at least 2 GB of RAM. More intensive use requires a multi-core CPU and at least 4 GB of RAM.
+- A virtual environment created with [uv](https://docs.astral.sh/uv/pip/environments)
 
-</TabItem>
-</Tabs>
+<details>
+<summary>Need help with virtual environments?</summary>
 
-To install a specific version of the Langflow package, add the required version to the command.
-<Tabs groupId="package-manager">
-<TabItem value="uv" label="uv" default>
+Virtual environments ensure Langflow is installed in an isolated, fresh environment.
+To create a new virtual environment, do the following.
 
-```bash
-uv pip install langflow==1.3.2
+<Tabs groupId="os">
+  <TabItem value="macOS" label="macOS">
+    1. Navigate to where you want your virtual environment to be created, and create it with `uv`.
+Replace `VENV_NAME` with your preferred name for your virtual environment.
+```
+uv venv VENV_NAME
+```
+2. Start the virtual environment.
+```
+source VENV_NAME/bin/activate
 ```
+Your shell's prompt changes to display that you're currently working in a virtual environment.
+```
+(VENV_NAME) ➜  langflow git:(main) ✗
+```
+3. To deactivate the virtual environment and return to your regular shell, type `deactivate`.
+To delete the virtual environment, type `rm -rf VENV_NAME`.
+  </TabItem>
```

Comment:
Take a look at your tab structure. Instructions for both OS types appear on each tab.